### PR TITLE
Update `about us` page

### PR DESCRIPTION
**Pull request by mribeirodantas**
This PR seeks to update the [About us](https://www.nextflow.io/about-us.html) page in the website.
- Improves slightly the wording in _About Nextflow_
- Updates info about individuals that are mentioned

### CODE REVIEW

File: content/about-us.html [hunk 1]
```diff
@@ -11,7 +11,7 @@ <h3>About Nextflow</h3>
                 <div class="about">
                     <div class="about-text" >
                         <p class="text-muted lh">
-                            Nextflow is free open source software distributed under the Apache 2.0 licence developed by <a href="http://www.seqera.io" target="_blank">Seqera Labs.</a>
+                            Nextflow is a free and open source software distributed under the Apache 2.0 licence and developed by <a href="http://www.seqera.io" target="_blank">Seqera Labs.</a>
```
Comment (pditommaso) [inline content/about-us.html:14]:
```suggestion
                            Nextflow is a free and open-source software distributed under the Apache 2.0 licence, developed by <a href="http://www.seqera.io" target="_blank">Seqera Labs.</a>
```

**Comment by pditommaso** [comment]
I think this also can go 

```
The project was started in the Notredame Lab at the <a href="http://www.crg.eu/" target="_blank">Centre for Genomic Regulation (CRG)
```

File: content/about-us.html (hunk 1 above)
Comment (mribeirodantas) [reply content/about-us.html:14]: What do you think of:

> Nextflow is a free and open-source software developed by Seqera Labs, and distributed under the Apache 2.0 license

?

**Comment by pditommaso** [comment]
Yes

File: content/about-us.html (hunk 1 above)
Comment (pditommaso) [reply content/about-us.html:14]: Think the commit proposal is preferable

**Pull request merged**